visualization/app/codeCharta/ui/ribbonBar/areaSettingsPanel/areaSettingsPanel.component.spec.ts

```diff
@@ -14,17 +12,6 @@ describe("AreaSettingsPanelComponent", () => {
 		})
 	})
 
-	it("should reset default margin checkbox on changes of margin slider", async function () {
```

Comment:
I guess this test is an open todo?

Reply:
It is, I would be up for fixing it, if we can use the material ui checkbox harness (As far as I know, this needs access to Angular's internal testing library)
The problem with this test is that we use queryByRole to get the checkbox, but that doesn't seem to work any more.

Reply:
This test now fails for two reasons:

First of all, because the input was removed. 
![image](https://user-images.githubusercontent.com/3596742/221702743-2fbf0c39-7804-4737-a4d7-bfe167f8bd11.png)
To be honest - I think this checkbox is useless, as we can reset all settings. I wouldn't mind to remove it to keep the UI lean - @ce-bo  what do you think?

Second we need to make the selector in line 21 more precise, as the slider already has an input and the querySelector finds the first occurence. We could e.g. change to `const marginInput = container.querySelector("input[type=number]")`

Reply:
@shaman-apprentice Oh yeah, I removed that because it never made sense to me. I made resetting the settings the same for all menus (a reset button at the end of the menu). Already discussed removing the checkbox with @ce-bo 👍 

I will look into the second point

Reply:
ah, when we remove the checkbox, we also need to remove the test for the checkbox. So it makes sense :) I think we should add a generic changelog entry like "Rework UI Design"